quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliVersionIT.java

```diff
@@ -17,13 +18,15 @@
 @Tag("quarkus-cli")
 @QuarkusScenario
 @DisabledOnQuarkusVersion(version = "1\\..*", reason = "Quarkus CLI has been reworked in 2.x")
+@DisabledOnQuarkusVersion(version = ".*redhat.*", reason = "Do not run CLI version check on productized bits")
 @DisabledIfSystemProperty(named = "profile.id", matches = "native", disabledReason = "Only for JVM verification")
 public class QuarkusCliVersionIT {
 
     @Inject
     static QuarkusCliClient cliClient;
 
     @Test
+    @Disabled("Latest CLI is supposed to be used, disabling for 2.2 branch")
```

Comment:
We already disabled this in class annotation.

Reply:
This disable is relevant for community runs, otherwise this would fail (2.2.5 based Quarkus, 2.7.0 vased CLI)

Reply:
Then maybe it's a better idea to remove this method completely?

Reply:
I think it's good to have it here for the future reference, e.g. when we create 2.7 branch